src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -346,13 +348,20 @@ def run(self, batch_inputs):
                 self._nlines,
             ),
         ):
-            self._pool.starmap(
-                self._timeout_process_wrapper,
-                [
-                    (run_start_time, self._inputs_queue, self._line_timeout_sec, result_list)
-                    for _ in range(self._n_process)
-                ],
-            )
+            try:
+                self._pool.starmap(
+                    self._timeout_process_wrapper,
+                    [
+                        (run_start_time, self._inputs_queue, self._line_timeout_sec, result_list)
```

Comment:
Please make sure user error would not be caught and reraised here.

Reply:
If an error is reported during execution, we wrap error and put it in run_info